Basic/Invaders/Assets/Scripts/PlayerControl.cs

```diff
@@ -192,17 +189,18 @@ private void InGameUpdate()
         if (Input.GetKeyDown(KeyCode.Space)) ShootServerRPC();
     }
 
-    [ServerRpc]
+    [Rpc(SendTo.Server)]
     private void ShootServerRPC()
```

Comment:
I'm sure you saw the alignment on the naming going forward for Rpcs here:
https://github.com/Unity-Technologies/com.unity.multiplayer.samples.coop/pull/883#discussion_r1531301114.

I think just a few still have the trailing ServerRPC or ClientRPC that we can remove. RPC -> Rpc as well.

Reply:
Oh, I though I changed them all, but definitely forgot a bunch, I'll double check!

Reply:
Fixed in 1c9ea1e